acceptance-tests/test-plugins/src/test/java/org/hyperledger/besu/services/BesuPluginContextImplTest.java

```diff
@@ -158,9 +172,68 @@ public void lifecycleExceptions() throws Throwable {
     assertThatExceptionOfType(IllegalStateException.class).isThrownBy(contextImpl::stopPlugins);
   }
 
-  private Optional<TestPicoCLIPlugin> findTestPlugin(final List<BesuPlugin> plugins) {
+  @Test
+  public void shouldRegisterAllPluginsWhenNoPluginsOption() {
+    final PluginConfiguration config = createConfigurationForAllPlugins();
+
+    assertThat(contextImpl.getRegisteredPlugins()).isEmpty();
+    contextImpl.registerPlugins(config);
+    final Optional<TestPicoCLIPlugin> testPluginOptional =
+        findTestPlugin(contextImpl.getRegisteredPlugins(), TestPicoCLIPlugin.class);
+    Assertions.assertThat(testPluginOptional).isPresent();
```

Comment:
nit: for uniformity always the static method imported, there also other occurrences below

Reply:
done